pom.xml

```diff
@@ -421,6 +421,7 @@
                             <rng>XML_STYLE</rng>
                             <spring.factories>CAMEL_PROPERTIES_STYLE</spring.factories>
                             <spring.provides>CAMEL_PROPERTIES_STYLE</spring.provides>
+<tm>CAMEL_PROPERTIES_STYLE</tm>
```

Comment:
Just a nit - can you fix up the indentation.

Reply:
@jamesnetherton this line shouldn'e be there atll, my ide didn't show it. I'll remove it

Reply:
tthanks for noticing this. This line is a part of stringtemplate extension (of course with proper indentation)